packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -70,16 +71,20 @@ export const announceStore = async ({
  *
  * @param runtimeConfiguration
  */
-export const announceClient = async (runtimeConfiguration: RuntimeConfiguration): Promise<void> => {
+export const announceClient = (runtimeConfiguration: RuntimeConfiguration): Promise<void> => {
   const { openIdConnect = {} } = runtimeConfiguration
 
   if (!openIdConnect.dynamic) {
     return
   }
 
-  const { client_id: clientId, client_secret: clientSecret } = await registerClient(openIdConnect)
-  openIdConnect.client_id = clientId
-  openIdConnect.client_secret = clientSecret
+  const promiseRegistration = registerClient(openIdConnect)
```

Comment:
this also is a step that must happen before continue the code flow, if we try to use the client on another place it can error

Reply:
Can you point me where we might need it? I was trying to understand the dependencies, but I got the impression that we have 2 clients: 1 for OIDC and another one which is the owncloud-sdk. So I had the impression that this particular client was only being used after `bootstrap` finishes, which still awaits fot this client registration to happen.